x/programs/examples/simulator/custom_imports/README.md

```diff
@@ -0,0 +1,3 @@
+# Custom import modules
+
+# TODO: have a containerized example that allows local imports to be enabled.
```

Comment:
This will be so cool for testing

Reply:
going to be a followup

Reply:
https://github.com/ava-labs/hypersdk/issues/551